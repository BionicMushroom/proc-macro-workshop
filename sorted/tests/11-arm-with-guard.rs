#[sorted::check]
fn main() {
    let x = 10;
    let y = 5;

    #[sorted]
    match x {
        0 if y == 5 => println!("first"),
        _ => println!("second"),
    }
}
