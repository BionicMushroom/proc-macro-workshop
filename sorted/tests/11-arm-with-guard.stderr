error: match arms with guards are unsupported by #[sorted]
 --> tests/11-arm-with-guard.rs:8:11
  |
8 |         0 if y == 5 => println!("first"),
  |           ^^
